rootston/xcursor.c

```diff
@@ -1,6 +1,85 @@
-#include <wlr/types/wlr_cursor.h>
+#define _POSIX_C_SOURCE 200809L
```

Comment:
How much of this can you bring into wlroots?

Reply:
This would require to create another data structure alongside `wlr_xcursor_theme` and `wlr_xcursor`.

Do we want this feature in wlroots? I'm still not sure about this. If we do it, I would prefer a separate type (`wlr_xcursor_manager`?) that would bridge the gap between `wlr_xcursor` and `wlr_cursor`, to make sure to keep these two separate. How does that sound?

Reply:
+1 to put it in wlroots. I like that name, too.

Reply:
I think we should try to make the interface to rootston as simple as possible.